bindings/pyroot/pythonizations/python/ROOT/_facade.py

```diff
@@ -9,13 +9,186 @@
 from cppyy import cppdef, include
 from libROOTPythonizations import gROOT
 from cppyy.gbl import gSystem
+import cppyy
 import cppyy.ll
 
 from ._application import PyROOTApplication
 from ._numbadeclare import _NumbaDeclareDecorator
 
 from ._pythonization import pythonization
 
+byteswap = ""
+
+
+def _CheckEndianessFromTypestr(typestr):
+    import ROOT
+
+    global byteswap
```

Comment:
No `global` please

Reply:
I got rid of it and made it a data member of the facase (for now :))